lib/content-services/src/lib/aspect-list/aspect-list.component.spec.ts

```diff
@@ -136,9 +136,8 @@ describe('AspectListComponent', () => {
         });
 
         it('should show the loading spinner when result is loading', async () => {
-            const delayResult = of(null).pipe(delay(0));
-            spyOn(nodeService, 'getNode').and.returnValue(delayResult);
-            spyOn(aspectListService, 'getAspects').and.returnValue(delayResult);
+            spyOn(nodeService, 'getNode').and.returnValue(of(null).pipe(delay(0)));
+            spyOn(aspectListService, 'getAspects').and.returnValue(of([]).pipe(delay(0)));
```

Comment:
Does adding delay(0) have any difference as opposed to not having it at all? I would assume the default value for delay is 0 anyways

Reply:
done